.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2.1
+jobs:
+  test:
+    parameters:
+        docker_image:
+          type: string
+          default: cimg/node:current-browsers
+    docker:
+      - image: << parameters.docker_image >>
+    steps:
+      - checkout
+      - run:
+          name: Setup
+          command: |
+            rm -rf node_modules package-lock.json
+            npm install
+      - run:
+          name: Running all unit tests
+          command: |
+            node -v
+            npm -v
+            npm run test
+
+workflows:
+  version: 2
+  test-all-node-versions:
+    jobs:
+      - test:
+          docker_image: circleci/node:14-browsers
+      - test:
+          docker_image: circleci/node:16-browsers
```

Comment:
Should test on later versions of node as well. Also, need to use the "modern" circleci images, as eventually, these old ones are going away. See https://github.com/iLib-js/ilib-lint-common/blob/main/.circleci/config.yml as an example